pkg/registry/admissionregistration/validatingadmissionpolicybinding/authz.go

```diff
@@ -85,26 +85,31 @@ func (v *validatingAdmissionPolicyBindingStrategy) authorize(ctx context.Context
 		}
 	}
 
-	paramRef := binding.Spec.ParamRef
-
-	// require that the user can read (verb "get") the referred resource.
-	attrs := authorizer.AttributesRecord{
-		User:            user,
-		Verb:            "get",
-		ResourceRequest: true,
-		Name:            paramRef.Name,
-		Namespace:       paramRef.Namespace,
-		APIGroup:        apiGroup,
-		APIVersion:      apiVersion,
-		Resource:        resource,
-	}
+	if paramRef := binding.Spec.ParamRef; paramRef.ParamKind == admissionregistration.BindingParamKindClusterWide && paramRef.ClusterWide != nil {
+		globalParamRef := paramRef.ClusterWide
+
+		//!TODO: can't perform static check on the namespaceParamRef
+		// should we copy this to the admission controller?
```

Comment:
Here is the authz check to make sure author has the right permission to the GKV they specified while creating binding. Could we do the similar check for per namespace paramRef? cc @jiahuif

Reply:
context for original conversation on this line:

https://github.com/kubernetes/kubernetes/pull/118644#discussion_r1258872438

Reply:
added an authz check here to ensure LIST across all namespaces for the given resource by @deads2k recommendation. and added test